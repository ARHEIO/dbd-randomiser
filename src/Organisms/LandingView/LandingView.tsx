import React from 'react';
import './LandingView.scss';
import { Link } from 'react-router-dom';

const LandingView = () => {
  return (
    <div className="landing-container">
       <Link className="link-button" to={`/survivor`}>
          <img alt="survivor" src="https://dqr3pglopijar.cloudfront.net/assets/iconHelpLoading_survivor.png" />
          <p>Randomise survivor build</p>
        </Link>

        <Link className="link-button" to={`/killer`}>
          <img alt="killer" src="https://dqr3pglopijar.cloudfront.net/assets/iconHelpLoading_killer.png" />
          <p>Randomise killer build</p>
        </Link>
    </div>
  );
}

export default LandingView;
