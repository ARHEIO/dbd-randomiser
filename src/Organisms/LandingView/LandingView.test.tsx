import React from 'react';
import { render } from '@testing-library/react';
import pretty from 'pretty';
import { MemoryRouter } from 'react-router-dom';
import LandingView from './LandingView';

describe('Landing View', () => {
  it('should render', () => {
    const { container } = render(
      <MemoryRouter>
        <LandingView />
      </MemoryRouter>,
    );
    expect(pretty(container.innerHTML)).toMatchSnapshot();
  });
});
