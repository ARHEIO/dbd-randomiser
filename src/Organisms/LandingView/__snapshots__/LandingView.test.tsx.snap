// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing View should render 1`] = `
"<div class=\\"landing-container\\"><a class=\\"link-button\\" href=\\"/survivor\\"><img alt=\\"survivor\\" src=\\"https://dqr3pglopijar.cloudfront.net/assets/iconHelpLoading_survivor.png\\">
    <p>Randomise survivor build</p>
  </a><a class=\\"link-button\\" href=\\"/killer\\"><img alt=\\"killer\\" src=\\"https://dqr3pglopijar.cloudfront.net/assets/iconHelpLoading_killer.png\\">
    <p>Randomise killer build</p>
  </a></div>"
`;
